daemon/api.go

```diff
@@ -919,41 +919,9 @@ func appIconGet(c *Command, r *http.Request) Response {
 	return iconGet(name, origin)
 }
 
-// plugConnection holds the identification of a slot that has been connected to a plug.
-type plugConnection struct {
-	Snap string `json:"snap"`
-	Name string `json:"slot"` // This is the slot name
-}
-
-// plugInfo holds details for a plug as returned by the REST API.
-type plugInfo struct {
-	Snap        string           `json:"snap"`
-	Name        string           `json:"plug"`
-	Interface   string           `json:"interface"`
-	Label       string           `json:"label"`
-	Connections []plugConnection `json:"connections"`
-}
-
-// getPlugs returns a response with a list of all the plugs and which slots use them.
-func getPlugs(c *Command, r *http.Request) Response {
-	var plugs []plugInfo
-	for _, plug := range c.d.interfaces.AllPlugs("") {
-		var slots []plugConnection
-		for _, slot := range c.d.interfaces.PlugConnections(plug.Snap, plug.Plug) {
-			slots = append(slots, plugConnection{
-				Snap: slot.Snap,
-				Name: slot.Slot,
-			})
-		}
-		plugs = append(plugs, plugInfo{
-			Snap:        plug.Snap,
-			Name:        plug.Plug,
-			Interface:   plug.Interface,
-			Label:       plug.Label,
-			Connections: slots,
-		})
-	}
-	return SyncResponse(plugs)
+// getInterfaces returns a response with a list of all the plugs and slots and their connections.
```

Comment:
`"returns a response with a list of all the plugs and slots"` => `"returns all plugs and slots"`